source/website/src/views/Step3.vue

```diff
@@ -233,6 +233,7 @@ SPDX-License-Identifier: Apache-2.0
           { value: 'DIMENSION', text: 'Dimension', disabled: false},
           { value: 'METRIC', text: 'Metric', disabled: false},
           { value: 'isMainEventTime', text: 'MainEventTime', disabled: false},
+          { value: 'LiveRamp', text: 'LiveRamp', disabled: false},
```

Comment:
This would be more consistent with how live ramp is described in the AMC upload docs:

{ value: 'LiveRamp ID', text: 'ramp_id disabled: false},

Reply:
Resolved